tokens/README.md

```diff
@@ -12,7 +12,7 @@ Send tokens to the recipients in `<BIDS_CSV>`.
 Example bids.csv:
 
 ```text
-primary_address,bid_amount_dollars
```

Comment:
The history here is that solana-tokens was originally implemented to do the Dutch auction distribution using their unmodified CSV files. Instead of trying to clean here, how about removing support for Bids entirely?

Reply:
Done: https://github.com/solana-labs/solana/pull/12253/commits/e1d2f05dfe7b8a5515efc5728428a4c56655cd5a